deployment_report/model/utils/test/test_viya_deployment_report_utils.py

```diff
@@ -52,11 +52,12 @@ def gathered_resources() -> Dict:
     # create a list of resource kinds to gather
     # nodes and networking kinds do not typically have owning objects, so these need to be called individually
     kinds_list: List = [
+        KubernetesResource.Kinds.CONFIGMAP,
```

Comment:
Is the order of this list arbitrary? If so alphabetical is fine, just making sure this is ok